build/scripts/generate-badges.ps1

```diff
@@ -0,0 +1,150 @@
+# Script for generating our badges line in the README.md
+
+[CmdletBinding(PositionalBinding=$false)]
+param ()
+
+$branchNames = @(
+    'master',
+    'master-vs-deps',
+    'dev16.0-preview2',
+    'dev16.0-preview2-vs-deps')
+
+function Get-AzureBadge($branchName, $jobName, $configName) {
+    $template = "[![Build Status](https://dev.azure.com/dnceng/public/_apis/build/status/dotnet/roslyn/roslyn-CI?label=build&branchname=$branchName&jobname=$jobName&configuration=$configName)]"
```

Comment:
It's not really a template if you substitute inline

Reply:
There's nothing that says a template has to be configurable just that it's a pattern https://www.merriam-webster.com/dictionary/template

👅